app/templates/modal.html

```diff
@@ -1,9 +1,9 @@
-<section ng-controller="ModalInstanceCtrl as modalInstance" class="modal-display">
+<section class="modal-display">
   <h2>Create New Room</h2>
   <p>Enter a room name</p>
   <form>
     <input class="input-style" type="text"><br>
-    <button class="cancel-button-style" ng-click="modalInstance.cancelModal()"><span class="cancel-button-text">Cancel</span></button>
-    <button class="create-button-style" ng-click="modalInstance.newRoom()"><span class="create-button-text">Create Room</span></button>
+    <button class="cancel-button-style" ng-click="modal.cancelModal()"><span class="cancel-button-text">Cancel</span></button>
```

Comment:
Needed to be modal, because that's what line 9 of `ModalCtrl` calls the `ModalInstanceCtrl`